src/components/POIForm.component.js

```diff
@@ -187,4 +170,46 @@ class POIForm extends Component {
   }
 }
 
+function FieldGroup({
+  controlId, //indentifier
+  label,
+  inputType, //text vs. textarea vs. file
+  helperText,
+  thisValue,
+  thisChange
+}) {
+  var fieldGroupModule
+
+  if (inputType === 'text') {
+    fieldGroupModule = (
+      <FormControl
+        type="text"
+        placeholder={helperText}
+        value={thisValue}
+        onChange={thisChange}
+      />
+    )
+  } else if (inputType === 'textarea') {
+    fieldGroupModule = (
+      <FormControl
+        componentClass="textarea"
+        placeholder={helperText}
+        value={thisValue}
+        onChange={thisChange}
+      />
+    )
+  } else if (inputType === 'file') {
+    fieldGroupModule = <FormControl type="file" placeholder={helperText} />
+  }
+
+  return (
+    <FormGroup controlid={controlId}>
+      <Col componentClass={ControlLabel} sm={2}>
```

Comment:
For consistency and ease of determining the column size, let's put the `sm` prop first. Change that everywhere else too